server/broadcast.go

```diff
@@ -346,12 +356,28 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 
 	// send segment to the orchestrator
 	glog.V(common.DEBUG).Infof("Submitting segment nonce=%d manifestID=%s seqNo=%d orch=%s", nonce, cxn.mid, seg.SeqNo, sess.OrchestratorInfo.Transcoder)
+	if sess.Sender != nil {
+		err := sess.Sender.ValidateTicketParams(pmTicketParams(sess.OrchestratorInfo.TicketParams))
```

Comment:
Since `err` isn't used outside of this code block I think we can do:

```golang
if err := sess.Sender.ValidateTicketParams(...); err != nil { ... }
```

for brevity since we only need to keep `err` around in the scope of this code block.